R/derive_param_wbc_abs.R

```diff
@@ -0,0 +1,162 @@
+#' Add a parameter for lab differentials converted to absolute values
+#'
+#' @description
+#'
+#' Add a parameter by converting lab differentials from percentage to absolute values
+#'
+#' @param dataset Input dataset
+#'
+#'   The variables specified by the `by_vars` argument, `PARAMCD`, and `AVAL`
```

Comment:
Does anything happen if PARAMCD or AVAL are not present?  If so, should we state what happens?

Reply:
Yes `PARAMCD` and `AVAL` are checked in the `assert_data_frame()` call, and if those variables are not in the input dataset, an error e.g. "Required variable `PARAMCD` is missing" will be issued. I think it's sufficient to state the variables are required, because if they are not users will see the error.